libnymea/libnymea.pro

```diff
@@ -6,6 +6,7 @@ TEMPLATE = lib
 QT += network bluetooth
 DEFINES += LIBNYMEA_LIBRARY
 
+QMAKE_CXXFLAGS += -Og
```

Comment:
Maybe we should add this only for debug? The recommended optimization level as I know is `-O2`

Reply:
right... thanks, will remove this again. it's a leftover from the abi-compliance-checker tests and not needed at this point. might come back later...